pure_pursuit_jerk_wrapper/src/pure_pursuit_jerk_wrapper.cpp

```diff
@@ -71,7 +71,7 @@ void PurePursuitJerkWrapper::trajectoryPlanHandler(const cav_msgs::TrajectoryPla
     wp.twist.twist.linear.x = speeds[i];
     ROS_DEBUG_STREAM("Setting waypoint idx: " << i <<", x: " << tp->trajectory_points[i].x << 
                             ", y: " << tp->trajectory_points[i].y <<
-                            ", speed: " << speeds[i]* 2.23694 << "mph" << "  Tactical plugin:"<< tp->trajectory_points[i].planner_plugin_name);
+                            ", speed: " << lag_speeds[i]* 2.23694 << "mph" << "  Tactical plugin:"<< tp->trajectory_points[i].planner_plugin_name);
```

Comment:
I think there might have been some git weirness here. speeds[i] is correct for this file, but lag_speeds[i] is correct for the regular wrapper.